app/frontend/packs/users/show.js

```diff
@@ -0,0 +1,48 @@
+let preview_avatar = document.getElementById('avatar-preview');
+let preview_avatar_src = preview_avatar.src;
+
+document.getElementById('user_image').addEventListener('change', (event) => {
+  if (validateFile(preview_avatar)) {
+    let pictureURL = URL.createObjectURL(event.originalTarget.files[0]);
+
+    preview_avatar.src = pictureURL;
+  }  else {
+    preview_avatar.src = preview_avatar_src;
+  }
+});
+
+function validateFile(picture){
+  const allowedExtensions =  ['jpg', 'jpeg', 'gif', 'png', 'svg'],
+        sizeLimit = 1000000; // 1 megabyte
```

Comment:
1 megabyte is `1024*1024` aka `1048576`
```suggestion
        sizeLimit = 1048576; // 1 megabyte
```

Reply:
sorry, didn't check out it when used, resolved)